src/components/AddNote/AddNote.tsx

```diff
@@ -30,12 +30,12 @@ export const AddNote = () => {
         id: uuidv4(),
         title: noteDetails.title,
         noteText: noteDetails.noteText,
-        dateCreated: new Date(),
+        dateCreated: new Date().getDate(),
         isArchived: false,
         isInTrash: false,
         bgColor: "white",
         labels: [],
-        updatedAt: new Date(),
+        updatedAt: new Date().getDate(),
```

Comment:
- You are storing just the `Day` here and not the date. 
- This will prevent the sorting from being accurate, as notes created on the same day will be sorted by `id` internally. 
- You can instead store the timestamp in milliseconds using `Date.now()`

Reply:
Thank you for pointing this. I changed `getDate()` with `getTime()

Reply:
Clean code and a great job overall. 👍